CMakeLists.txt

```diff
@@ -34,6 +34,32 @@ endif()
 
 find_ros1_package(std_msgs REQUIRED)
 
+# find all known ROS1 message/service packages
+find_ros1_package(rosmsg REQUIRED)
+execute_process(
+    COMMAND bash -c "rosmsg list"
```

Comment:
I previously tried  just the command and it resulted in an empty string that's why I added the `bash -c` here. Did investigate the why though